src/validators/validateDependencies.ts

```diff
@@ -13,7 +13,7 @@ const isUnpublishedVersion = (version: string): boolean => {
 			version,
 		) ||
 		// https://pnpm.io/next/workspaces#workspace-protocol-workspace
-		/^workspace:((\^|~)?[0-9.x]*|(<=?|>=?)?[0-9.x][\-.+\w]+|\*)?$/.test(
+		/^workspace:([a-z0-9@/][\w@/.\-]*@)?((\^|~)?[0-9.x]*|(<=?|>=?)?[0-9.x][\-.+\w]+|\*)?$/.test(
```

Comment:
🤔 I wonder if we should just allow anything after `workspace:` at this point, like we do with `npm:` the next condition down, rather than trying to craft a super specific regex?

Reply:
Yeah 😬 this is looking very difficult to maintain. And honestly already was.